### PR TITLE
Some cleanups.

### PR DESCRIPTION
**Pull request by MFDGaming**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -11,49 +11,39 @@
...
-
-        self.loaded_plugins:list = []
-        
-        
+        self.plugin_folder_path: str = os.path.join(os.getcwd(), "plugins")
+        self.loaded_plugins: list = []
         
     def on_load(self):
-        self.logger.info("DevTools is on fire!!!")
+        self.server.logger.info("DevTools is on fire!!!")
```
Comment (HunterAzar) [inline main.py]: look at new commit

Comment (MFDGaming) [reply main.py]: yes

Comment (MFDGaming) [reply main.py]: reinitializing a new logger is pointless

**Comment by HunterAzar** [comment]
Thank you @MFDGaming

**Pull request merged**